### ISSUE TITLE
To consider: policy for maintining without largely adding features.

### ISSUE THREAD

**Issue by mikeshardmind** [mikeshardmind/SinbadCogs#212]
There's not much which is going to break any time soon with this, so I may just write out a policy for how that can be handled, and minimally support the existing feature set from here out.

**Comment by mikeshardmind** [comment]
The policy will go up after I've finished work on [a different, more performant means of storing data without needing to convert existing data to be fully relational ](https://github.com/mikeshardmind/rapid_dev_storage) and decoupled the core behavior of the cogs to anything red specific.

the setup function will then behave differently for Red and Non-Red instances choosing whichever command base class and cog base makes sense.

This will allow me to work in parallel on other things, so new features will likely be back on the table again later on.

**Comment by DiscordLiz** [comment]
Why bother? It's been forked by people who still use Red, and you can focus on making the cogs better without needing that constraint.

I just see another set up for disappointment here. People in the red server want features, they don't care who provides them, and they don't appreciate the work or the thought and consideration that have gone into the tools you've made. It's a thankless job that involves you having to play within the specific way Red does certain things when if you weren't tied into that idea, you'd have a lot more ability to improve the cogs.

I'm 100% behind your idea of just replacing Red with something more suited to what we need, but I want a reason why you're going to keep supporting the Red users if you do that. If you don't have one that I can agree is worth it, I'm going to have to ask that you no longer ask me to look over changes you make here, and you're going to be on your own with it.

**Comment by mikeshardmind** [comment]
It's complicated. I saw how a couple of issues were handled on forks and the ban sync change that needed to be made to be able to offer our usage to a few other "friendly" servers. Seeing that I decided that since (at the time of making this issue, not that this still holds) we were still using red (albeit, somewhat begrudgingly) to avoid extra work that I could easily just publish the fixes I already was using locally despite that officially I had dropped support.

I don't know that I completely agree with your assessment of it being entirely thankless work, but there definitely are plenty of users who fall into that category.

I don't think you'll find those as compelling reasons to put that effort into it, as little effort as it might be, as once I wrote it out I don't either anymore. I'm closing this, and won't continue this line of thought. There will eventually be more drastic changes in the repo which likely won't be Red compatible.
